habitat_baselines/rl/models/simple_cnn.py

```diff
@@ -0,0 +1,138 @@
+import numpy as np
```

Comment:
Maybe all baselines can share model architectures and move it to `habitat_baselines/models`.

Reply:
Not all baselines use DL tho